src/Test.Utilities.Sarif/HttpMockHelper.cs

```diff
@@ -20,6 +20,11 @@ public static StringContent AnyContent()
             return new StringContent(AnyContentText);
         }
 
+        public static HttpResponseMessage CreateOKResponse()
```

Comment:
>CreateOKResponse [](http://example.com/codeflow?start=42&length=16)

So, this code illustrates some of the dangers of statics. There is no such thing as a static response, once you read the stream from it, that stream is disposed.

I'm surprised we haven't gotten burned by this before. We need to revisit this test class. I took a small change along the lines of what I think we'll need, helpers that dynamically create new instances of the default data.

Though honestly, a helper that wraps a single line of code may not be worth implementing at all. :)